### PR TITLE
Linux base CI

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

**Comment by reuben** [comment]
All green on my side: https://github.com/reuben/STT/actions/runs/773645770

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -25,33 +25,584 @@ jobs:
...
+          ref: ${{ env.swig_hash }}
+      - run: |
+          mkdir -p build-static/
+      - uses: actions/cache@v2
+        id: swig-build-cache
+        with:
+          path: build-static/
+          key: swig-2-${{ runner.os }}-${{ env.swig_hash }}
+      - run: |
+          sudo apt-get install -y --no-install-recommends autoconf automake bison build-essential mingw-w64
```
Comment (lissyx) [inline .github/workflows/build-and-test.yml:87]: dont need `mingw64` here

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -25,33 +25,584 @@ jobs:
...
+        with:
+          repository: "swig/swig"
+          ref: ${{ env.swig_hash }}
+      - run: |
+          mkdir -p build-static/
+      - uses: actions/cache@v2
+        id: swig-build-cache
+        with:
+          path: build-static/
+          key: swig-2-${{ runner.os }}-${{ env.swig_hash }}
```
Comment (lissyx) [inline .github/workflows/build-and-test.yml:85]: what's with the `2` ?

Comment (reuben) [reply .github/workflows/build-and-test.yml:85]: Had a broken build that got cached and needed to bump it.

File: .github/workflows/build-and-test.yml [hunk 3]
```diff
@@ -25,33 +25,584 @@ jobs:
...
+          name: "libdeepspeech.${{ matrix.build-flavor }}.zip"
+          path: ${{ github.workspace }}/artifacts/libdeepspeech.zip
+  build-python-Linux:
+    name: "Lin|Build Python bindings"
+    runs-on: ubuntu-20.04
+    needs: [ build-lib_Linux, swig_Linux ]
+    strategy:
+      matrix:
+        build-flavor: ["tf", "tflite"]
+        python-version: [3.6.13, 3.7.10, 3.8.9, 3.9.4]
```
Comment (lissyx) [inline .github/workflows/build-and-test.yml:313]: nit: it looks like `setup-python` hits more frequently api rate limit when requesting finer grained version than just X.Y

**Review by lissyx — APPROVED** [review]

**Pull request merged**